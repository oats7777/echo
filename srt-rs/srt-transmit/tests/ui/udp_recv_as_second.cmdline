["udp://:4004" ,  "udp://:4005"]