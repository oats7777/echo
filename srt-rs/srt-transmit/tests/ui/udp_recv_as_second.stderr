Invalid settings detected: Must designate a ip to send to to send UDP. Example: udp://127.0.0.1:1234 ,  not udp://:1234

See srt-transmit --help for more info
